journal/original/12-chapter-10.adoc

```diff
@@ -1,7 +1,9 @@
 == Chapter X.
 
+[.blurb]
 An Epistle to the Women`'s Half-Yearly Meeting, 1677.
```

Comment:
You forgot to put the three equal signs === before this blurb heading. 🙂

Reply:
I can't believe I missed this in looking at the PDF!!!